api/src/main/resources/firecloud.yaml

```diff
@@ -2570,6 +2570,27 @@ components:
           enum:
             - NoNewCalls
             - ContinueWhilePossible
+        memoryRetryMultiplier:
```

Comment:
copied from latest Rawls

We only use this new field, but the others are added for completion